README.md

```diff
@@ -1,90 +1,149 @@
 # Graph Protocol Solidity Smart Contracts
-
+![License: MIT](https://img.shields.io/badge/license-MIT-green)
 ![Version Badge](https://img.shields.io/badge/version-1.6.0-lightgrey.svg)
 ![CI Status](https://github.com/graphprotocol/contracts/actions/workflows/npmtest.yml/badge.svg)
 [![codecov](https://codecov.io/gh/graphprotocol/contracts/branch/dev/graph/badge.svg?token=S8JWGR9SBN)](https://codecov.io/gh/graphprotocol/contracts)
+# The Graph Protocol Contracts
+[The Graph](https://thegraph.com/) is an indexing protocol for querying networks like Ethereum,
```

Comment:
There are some line breaks here that makes it look odd when looking the preview markdown. Maybe an external editor added those.

Reply:
updated